src/main/java/core/basesyntax/service/impl/ReportServiceImpl.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.dao.FruitTransactionDao;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.ReportService;
+import core.basesyntax.service.WriteScvService;
+
+public class ReportServiceImpl implements ReportService {
+    private FruitTransactionDao fruitTransactionDao;
+    private WriteScvService writeScvService;
+    private final String nameReportFile = "report.csv";
+    private StringBuilder data;
+
+    public ReportServiceImpl(FruitTransactionDao fruitTransactionDao,
+                             WriteScvService writeScvService) {
+        this.fruitTransactionDao = fruitTransactionDao;
+        this.writeScvService = writeScvService;
+    }
+
+    @Override
+    public void createReport() {
+        data = new StringBuilder("fruit,quantity\n");
+        for (FruitTransaction fruit : fruitTransactionDao.getAllListDb()) {
+            data.append(fruit.getFruit()).append(",").append(fruit.getQuantity()).append("\n");
```

Comment:
All magic numbers, strings, symbols in your code should be constants.
[this article](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names)

Reply:
not fixed